mac/hid.c

```diff
@@ -552,6 +552,35 @@ static struct hid_device_info *create_device_info_with_usage(IOHIDDeviceRef dev,
 		cur_dev->interface_number = -1;
 	}
 
+    if (cur_dev->interface_number == -1 || cur_dev->interface_number == 0) {
```

Comment:
`cur_dev->interface_number == 0` is a valid interface number
it doesn't look right to consider it for a fallback

Reply:
I cause this problem:  my device interface number is not 0. but the old code return 0 on macos 13.3.  on macos13.2，the old code works fine(not 0) .  so 0 may not be valid!